src/VecSim/algorithms/svs/svs_utils.h

```diff
@@ -102,18 +98,20 @@ makeVamanaBuildParameters(const SVSParams &params) {
     //      without compromising the quality of the graph.
     // * use_search_history (true): now: is enabled if not disabled explicitly
     //                              future: default value based on other index parameters
-    const auto construction_window_size = getOrDefault(params.construction_window_size, 200);
-    const auto graph_max_degree = getOrDefault(params.graph_max_degree, 32);
+    const auto construction_window_size = getOrDefault(params.construction_window_size, SVS_VAMANA_DEFAULT_CONSTRUCTION_WINDOW_SIZE);
+    const auto graph_max_degree = getOrDefault(params.graph_max_degree, SVS_VAMANA_DEFAULT_GRAPH_MAX_DEGREE);
 
     // More info about VamanaBuildParameters can be found there:
     // https://intel.github.io/ScalableVectorSearch/python/vamana.html#svs.VamanaBuildParameters
     return svs::index::vamana::VamanaBuildParameters{
-        getOrDefault(params.alpha, (params.metric == VecSimMetric_L2 ? 1.2f : 0.95f)),
+        getOrDefault(params.alpha, (params.metric == VecSimMetric_L2 ?
+            SVS_VAMANA_DEFAULT_ALPHA_L2 : SVS_VAMANA_DEFAULT_ALPHA_IP)),
         graph_max_degree,
         construction_window_size,
         getOrDefault(params.max_candidate_pool_size, construction_window_size * 3),
         getOrDefault(params.prune_to, graph_max_degree - 4),
-        params.use_search_history != VecSimOption_DISABLE
+        params.use_search_history == VecSimOption_AUTO ? SVS_VAMANA_DEFAULT_USE_SEARCH_HISTORY :
```

Comment:
isn't `VecSimOption_AUTO` is a valid configuration that in the future might not necessarily `SVS_VAMANA_DEFAULT_USE_SEARCH_HISTORY?`

i believe that it was intended to implement something like:
```
if (params.use_search_history == VecSimOption_AUTO) ? choose_best_search_history_config()
else  if (params.use_search_history == VecSimOption_ENABLE) true
else /* VecSimOption_DISABLE */ false
```

Reply:
right, but VecSimOption_AUTO is also the default value (0), and there is no `choose_best_search_history_config()` implentation yet (the default is always true)